### PR TITLE
Add support for K80 and A2

### PR DESCRIPTION
**Pull request by knikolla**

### CODE REVIEW

File: etc/flavors_cache.json [hunk 1]
```diff
@@ -334,5 +334,12 @@
         "vcpus": 2,
         "memory": 4096,
         "storage": 10
+    },
+    {
+        "id": 320,
```
Comment (joachimweyl) [inline etc/flavors_cache.json:339]: So that I understand better, this is just an arbitrary unique code?

File: etc/flavors_cache.json [hunk 2]
```diff
@@ -334,5 +334,12 @@
...
+        "name": "gpu-k80.1",
+        "vcpus": 8,
```
Comment (joachimweyl) [inline etc/flavors_cache.json:341]: 6

File: etc/flavors_cache.json [hunk 3]
```diff
@@ -334,5 +334,12 @@
...
+        "memory": 64512,
```
Comment (joachimweyl) [inline etc/flavors_cache.json:342]: 31744

File: etc/flavors_cache.json (hunk 2 above)
Comment (joachimweyl) [reply etc/flavors_cache.json:341]: a2 has that much cpu

File: etc/flavors_cache.json (hunk 3 above)
Comment (joachimweyl) [reply etc/flavors_cache.json:342]: a2 has that much RAM.

Comment (knikolla) [reply etc/flavors_cache.json:342]: This is what the flavor has on OpenStack, I'm just caching programmatically what I encounter.

```
knikolla@nessie ~ % openstack --os-cloud nerc flavor show gpu-k80.1
+----------------------------+--------------------------------------+
| Field                      | Value                                |
+----------------------------+--------------------------------------+
| OS-FLV-DISABLED:disabled   | False                                |
| OS-FLV-EXT-DATA:ephemeral  | 0                                    |
| access_project_ids         | ['<OMITTED>'] |
| description                | None                                 |
| disk                       | 20                                   |
| id                         | 3d8fac2a-6598-44a0-880f-1d360f4b6479 |
| name                       | gpu-k80.1                            |
| os-flavor-access:is_public | False                                |
| properties                 | pci_passthrough:alias='K80:1'        |
| ram                        | 64512                                |
| rxtx_factor                | 1.0                                  |
| swap                       |                                      |
| vcpus                      | 8                                    |
+----------------------------+--------------------------------------+
```

File: etc/flavors_cache.json (hunk 1 above)
Comment (knikolla) [reply etc/flavors_cache.json:339]: No, it's the primary key in the database. It's not exposed externally.

File: etc/flavors_cache.json (hunk 3 above)
Comment (joachimweyl) [reply etc/flavors_cache.json:342]: @Milstein & @jtriley please double check the K80 flavor on OpenStack. It looks like it was mixed up with A2 flavor.

**Comment by jtriley** [comment]
Also the new public k80 flavors were pushed out yesterday so we should pull the list again and rebuild that cache

**Review by naved001 — APPROVED** [review]

**Pull request merged**